ably/personal_task.go

```diff
@@ -75,7 +60,19 @@ func personalTask(testConfig TestConfig) {
 			}
 			defer sub.Close()
 
-			go reportSubscriptionToLocust(ctx, sub)
+			go func() {
+				for {
+					select {
+					case <-ctx.Done():
+						return
+					case msg := <-sub.MessageChannel():
+						timeElapsed := millisecondTimestamp() - msg.Timestamp
```

Comment:
This is the line of code that the reported stack trace points at (i.e. [`/opt/ably/ably/personal_task.go:41`](https://github.com/ably/ably-boomer/blob/master/ably/personal_task.go#L41)), won't the same issue exist with the code being here too?